src/sg/gov/moh/healthcert/1.0/interim-pdt-healthcert-schema.json

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
Do the issued PDTs have `notarisationMetadata`?

Reply:
The Notarised PDT HealthCert has it: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/example_notarized_healthcert_with_nric_unwrapped.json#L84-L88

As well as attachments: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/example_notarized_healthcert_with_nric_unwrapped.json#L90-L96

Reply:
Ok will define it